ui/apps/platform/src/Containers/NetworkGraph/common/NetworkGraphIcons.tsx

```diff
@@ -8,3 +8,7 @@ export function DeploymentIcon() {
 export function NamespaceIcon() {
     return <Badge style={{ backgroundColor: 'rgb(32,79,23)' }}>NS</Badge>;
 }
+
+export function CidrBlockIcon() {
+    return <Badge style={{ backgroundColor: 'rgb(0,0,0)' }}>IP</Badge>;
```

Comment:
Question that I forgot to ask earlier, about the colors of these "badges".

Is our UX team giving these colors, and will we try to get them added to PatternFly, so that all Red Hat apps can standardize on colors for k8s entities?

Reply:
I wasn't given color values, but I used what I saw from the mocks. I'm not entirely sure about standardizing these for other apps though. For the CIDR block icon, I just tried to make it similar to what we had already. I asked a question about where to get that IP icon. I'll wait on a response from the UX team about that. @alanonthegit @zhenpesky maybe you have thoughts on this

Reply:
Will merge but make changes as necessary based on UX's reply